specs/nfs-server-service.json

```diff
@@ -0,0 +1,23 @@
+{
+    "kind": "Service",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "nfs-server"
+    },
+    "spec": {
+        "ports": [
+            {
+                "name": "nfs",
+                "port": 2049
+            },
+            {
+                "name": "rpc",
+                "port": 111
+            }
+        ],
+        "selector": {
+            "role": "nfs-server"
+        },
+        "clusterIP": "10.247.126.77"
```

Comment:
clusterIp not external
this the simple way to make connect able by persistantvolume